examples/families-exemplars.rs

```diff
@@ -0,0 +1,47 @@
+use prometheus_client::{
```

Comment:
Thanks for also adding an example. I wonder whether people would easier discover this as a doc example on `HistogramWithExemplars` or `Family` (potentially one linking to the other). What do you think @adamchalmers?

Reply:
Sounds good, I'll remove this example and replace it with examples on the docs for CounterWithExemplar and HistogramWithExemplars.